src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,21 @@
+const CHECK_STATUS = 'categories/checkStatus';
+
+export const checkStatus = () => ({ type: CHECK_STATUS });
+
+const initialState = {
+  status: 'Under Construction',
+};
+
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return {
+        ...state,
+        status: 'Active',
+      };
+    default:
+      return state;
+  }
+};
+
```

Comment:
- **[OPTIONAL]** Your categories' actions and reducer should be set up using createSlice from the redux toolkit. This is a prerequisite for this project, but since the previous reviewer missed I will mark it as optional.